<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>50</horstretch>
     <verstretch>50</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt;
background-color: &quot;crimson&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
